UnitTests/Input/ResponderTests.cs

```diff
@@ -129,4 +130,124 @@ public void IsOverridden_True_IfOverridden ()
 		Assert.Empty (Responder.Instances);
 #endif
 	}
+
```

Comment:
Is there value in having a version of these unit tests that tests all view sub-classes?

Reply:
I think so, because I'm testing 3 cases:
1. Without disposed notification.
2. With disposed notification only on the second container.
3. With disposed notification only on the first container.

Reply:
Ak I think your talking about why not only using the base class `Responder`. The reason was I wanted to use `View` properties, like the `Id`, `Subviews`, etc...